bin/workflows/pycbc_make_coinc_search_workflow

```diff
@@ -155,7 +155,23 @@ precalc_psd_files = wf.setup_psd_workflow(workflow, analyzable_segs,
 bank_files = wf.setup_tmpltbank_workflow(workflow, analyzable_segs,
                                          datafind_files, output_dir="bank",
                                          psd_files=precalc_psd_files)
-hdfbank = wf.convert_bank_to_hdf(workflow, bank_files, "bank")
+
+# Determine the extension of the bank file taken as input
+cp = workflow.cp
+tmpltbankMethod = cp.get_opt_tags("workflow-tmpltbank", "tmpltbank-method", '')
```

Comment:
Seems a bit overkill to look at the configparser here. An easier idea is to simply check the output of the bank_files to see what the file extension is. You can get it from the logical file name. You should be able to get this from something like bank_files[0].name.